org.jacoco.core.test/src/org/jacoco/core/internal/instr/SafetySlotTest.java

```diff
@@ -0,0 +1,118 @@
+/*******************************************************************************
+ * Copyright (c) 2009, 2023 Mountainminds GmbH & Co. KG and Contributors
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Evgeny Mandrikov - initial API and implementation
+ *
+ *******************************************************************************/
+package org.jacoco.core.internal.instr;
+
+import org.jacoco.core.instr.Instrumenter;
+import org.jacoco.core.runtime.IRuntime;
+import org.jacoco.core.runtime.RuntimeData;
+import org.jacoco.core.runtime.SystemPropertiesRuntime;
+import org.jacoco.core.test.TargetLoader;
+import org.jacoco.core.test.validation.JavaVersion;
+import org.junit.Test;
+import org.objectweb.asm.ClassReader;
+import org.objectweb.asm.ClassWriter;
+import org.objectweb.asm.Label;
+import org.objectweb.asm.MethodVisitor;
+import org.objectweb.asm.Opcodes;
+
+public class SafetySlotTest {
+
+	@Test
+	public void original() throws Exception {
```

Comment:
As there is no assertion in the test maybe a descriptive test name can describe the purpose of the test:

```suggestion
	public void should_verify_original_class_without_errors() throws Exception {
```

Reply:
```suggestion
	public void jvm_should_verify_original_class_without_errors() throws Exception {
```